mlflow/telemetry/__init__.py

```diff
@@ -1,3 +1,4 @@
 from mlflow.telemetry.client import get_telemetry_client, set_telemetry_client
+from mlflow.telemetry.track import record_usage_event
```

Comment:
Could we revert this change as it's not related to this PR?
Also since set_telemetry_client is used when importing MLflow I think we shouldn't import too many thing in this file, do you feel this change is necessary?